dali/operators/image/color/color_twist.cu

```diff
@@ -34,15 +34,23 @@ bool ColorTwistGpu::SetupImpl(std::vector<OutputDesc> &output_desc, const Device
   const auto &input = ws.template Input<GPUBackend>(0);
   output_desc.resize(1);
   DetermineTransformation(ws);
+  auto sh = input.shape();
```

Comment:
As mentioned in the CPU case, this could be unified for both backends, or simplified

Reply:
Removed CallSetup.